ipaclient/install/client.py

```diff
@@ -2362,24 +2362,22 @@ def sync_time(options, fstore, statestore):
         ntp_servers = options.ntp_servers
 
     if ntp_servers:
-        if timeconf.configure_chrony(ntp_servers, options.ntp_pool,
-                                     fstore, statestore):
-            print("Done configuring chrony.")
-        else:
-            print("Warning: IPA was unable to sync time with chrony!")
-            print("         Time synchronization is required for IPA "
-                  "to work correctly")
-            logger.warning(
-                "Unable to sync time with chrony server, assuming the time "
-                "is in sync. Please check that 123 UDP port is opened, "
-                "and any time server is on network.")
+        timeconf.configure_chrony(ntp_servers, options.ntp_pool,
+                                  fstore, statestore)
     else:
         print("Warning: Skipping chrony configuration. "
```

Comment:
This doesn't read very well during an install. I don't know that this needs a warning, just an info stating "Using default chrony configuration" at best. Similar with log message, just drop the Skipping bit because it in fact **is** being configured.